src/core_plugins/input_control_vis/public/control/filter_manager/phrase_filter_manager.js

```diff
@@ -61,10 +61,22 @@ export class PhraseFilterManager extends FilterManager {
       return;
     }
 
-    return kbnFilters
+    const values = kbnFilters
       .map((kbnFilter) => {
         return this._getValueFromFilter(kbnFilter);
       })
+      .filter(value => {
+        if (typeof value === 'undefined' || value == null) {
```

Comment:
The entire filter could become this: `.filter(value => value != null)`, if you want to keep the `!=` instead of `!==`. Alternatively, it could be: `.filter(value => typeof value !== 'undefined' && value !== null)`.